Mastermind/MasterMind/Mastermind.cs

```diff
@@ -1,59 +1,86 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace Mastermind
 {
+
```

Comment:
Good job on Mastermind. You could have added in the hint that tells you how many of each letter you got right or almost right, but still a good job.